setup.md

```diff
@@ -15,14 +15,12 @@ If you’d prefer, you can set up GUAC with Kubernetes with the experimental
 Note that these helm charts are still experimental and are hosted in a
 third-party repo and may not be synchronized with the GUAC repo.
 
-GUAC consists of multiple components. You may have seen a subset of these used
-in various [GUAC demos]({{ site.baseurl }}{% link guac-use-cases.md %}). To get
-the most value out of GUAC, you’ll need to set up all components. This tutorial
-will walk you through how to deploy GUAC, using Docker Compose, so that you get
-the full set of components.
-
-If you’re curious about the various GUAC components and what they do, see [How
-GUAC components work together]({{ site.baseurl }}{%link guac-components.md %}).
+GUAC consists of multiple components. This tutorial will walk you through how to
```

Comment:
Could we rephrase this to make it sound more "approachable"? Meaning, I think we should remove this line entirely (have it as part of the "full" deployment guide). Here just say something like minimal components can be brought up to achieve the demos" (like in some cases, when we start doing "SbomDiff", we don't even need the collectorSub and the other collectors running).

Reply:
Good point! Updated.